core/src/main/java/org/frankframework/configuration/Configuration.java

```diff
@@ -221,6 +224,10 @@ public void configure() throws ConfigurationException {
 			}
 
 			configurableLifecycle.configure();
+
+			if (errorMessageFormatter instanceof IConfigurable configurableErrorMessageFormatter) {
+				configurableErrorMessageFormatter.configure();
+			}
```

Comment:
Als het een lifecycle zou hebben (en hij singelton is) wat volgens mij wel kan, hoeft dit niet.

Reply:
Maar het is geen singleton.
Ik had wel overwogen om de error formatters een lifecycle te geven maar dat leek me overkill.

Ik kan het alsnog uittesten als je wilt.